package.json

```diff
@@ -23,22 +23,25 @@
     "url": "https://github.com/Deschtex/html2react/issues"
   },
   "homepage": "https://github.com/Deschtex/html2react#readme",
+  "dependencies": {
+    "react-dom-core": "^0.0.2"
+  },
   "devDependencies": {
-    "babel-cli": "^6.10.1",
-    "babel-jest": "^19.0.0",
-    "babel-preset-es2015": "^6.9.0",
+    "babel-cli": "^6.26.0",
+    "babel-jest": "^21.2.0",
+    "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-1": "^6.5.0",
+    "babel-preset-stage-1": "^6.24.1",
     "in-publish": "^2.0.0",
-    "jest": "^19.0.2",
-    "react": "^15.4.0",
-    "react-dom": "^15.4.0",
-    "react-test-renderer": "^15.5.4",
-    "regenerator-runtime": "^0.10.3"
+    "jest": "^21.2.1",
+    "react": "^15.4.0 || ^16.0.0",
```

Comment:
How would having these two versions here work? For peer dependencies it makes sense, but for dev dependencies, I don't think so. Or?